### ISSUE TITLE
App not starting

### ISSUE THREAD

**Issue by partitect** [expo/expo#7840]
expo start correctly and expo dev server is tunnel ready but app dont start on ios simulator and android device.
waiting on splash screen and after “could not connect to development server”
this error only on my mac, windows 10 pc work normally

Sometimes when i start on web it can be work.

**Comment by cruzach** [comment]
Unfortunately this isn't enough information to help you troubleshoot this 🙁 I suggest checking out [this SO post](https://stackoverflow.com/questions/58458091/expo-project-error-could-not-connect-to-development-server-after-updating-expo-s), and trying the solution there. If you still can't figure it out, please provide everything we ask for in the bug report template in a new issue, good luck!
